.github/workflows/nightly.yaml

```diff
@@ -4,6 +4,11 @@ on:
     - cron: '0 0 * * Sun'
   workflow_dispatch:
 
+env:
+  REGISTRY: ghcr.io
+  IMAGE_NAME: weaveworks/wego-app
+  USE_NIGHTLY_BUILD_TAG: "true"
```

Comment:
One small nit. Can we have this as `1` and then just check that it's set? There is usually no point in doing a string `"true"` in bash. :)

Reply:
Oh right, you are already doing this:
```go
	} else if os.Getenv("USE_NIGHTLY_BUILD_TAG") != "" {
		version = "nightly-build"
	}
```

so just set it to `1`. :)